scalability_and_performance/low_latency_tuning/cnf-debugging-low-latency-tuning-status.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cnf-debugging-low-latency-tuning-status"]
+= Debugging low latency node tuning status
+include::_attributes/common-attributes.adoc[]
+:context: cnf-debugging-low-latency
+
+toc::[]
+
+Use the `PerformanceProfile` custom resource (CR) status fields for reporting tuning status and debugging latency issues in the cluster node.
+
+include::modules/cnf-debugging-low-latency-cnf-tuning-status.adoc[leveloffset=+1]
+
+include::modules/cnf-collecting-low-latency-tuning-debugging-data-for-red-hat-support.adoc[leveloffset=+1]
```

Comment:
Only on heading per module so should `About the must-gather tool` and `Gathering low latency tuning data` sections be brought to the assembly level?

Peer-review checklist 

![Screenshot from 2024-05-02 12-53-17](https://github.com/openshift/openshift-docs/assets/57954076/3fa96781-ad6d-452d-916b-d4bd65a79e52)

Reply:
This would be a significant rewrite so will add this to the content rework PR